### PR TITLE
update readme and changelog

### PR DESCRIPTION
**Pull request by Jimexist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,9 @@ Simple usage:
 
 ```python
 import datafusion
+from datafusion import functions as f
```
Comment (alamb) [inline README.md:42]: When I tried running this example now I get the following error:

```python
>>> # create a new statement
>>> df = df.select(
...     f.col("a") + f.col("b"),
...     f.col("a") - f.col("b"),
... )
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/Users/alamb/Software/virtual_envs/arrow_dev/lib/python3.9/site-packages/datafusion/functions.py", line 23, in __getattr__
    return getattr(functions, name)
AttributeError: module 'functions' has no attribute 'col'
>>> 
```

File: README.md [hunk 2]
```diff
@@ -125,6 +123,14 @@ pip install datafusion
 python -m pip install datafusion
 ```
 
+You can verify the installation by running:
+
+```text
```
Comment (alamb) [inline README.md:128]:
```suggestion
```python
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,111 +19,56 @@
...
+- use stable not nightly [\#11](https://github.com/datafusion-contrib/datafusion-python/pull/11) ([Jimexist](https://github.com/Jimexist))
+- ci: test against more compilers, setup clippy and fix clippy lints [\#9](https://github.com/datafusion-contrib/datafusion-python/pull/9) ([cpcloud](https://github.com/cpcloud))
+- Fix use of importlib.metadata and unify requirements.txt [\#8](https://github.com/datafusion-contrib/datafusion-python/pull/8) ([cpcloud](https://github.com/cpcloud))
+- Ship the Cargo.lock file in the source distribution [\#7](https://github.com/datafusion-contrib/datafusion-python/pull/7) ([cpcloud](https://github.com/cpcloud))
+- add \_\_version\_\_ attribute to datafusion object [\#3](https://github.com/datafusion-contrib/datafusion-python/pull/3) ([tfeda](https://github.com/tfeda))
+- fix ci by fixing directories [\#2](https://github.com/datafusion-contrib/datafusion-python/pull/2) ([Jimexist](https://github.com/Jimexist))
+- setup workflow [\#1](https://github.com/datafusion-contrib/datafusion-python/pull/1) ([Jimexist](https://github.com/Jimexist))
+
+## [0.5.0](https://github.com/datafusion-contrib/datafusion-python/tree/0.5.0) (2022-03-10)
```
Comment (alamb) [inline CHANGELOG.md:49]: something looks strange to me about these older version (in that it contains entries for the rcs)

**Review by alamb — COMMENTED** [review]
Looking better!

**Review by houqp — APPROVED** [review]

**Pull request merged**